diff -up vmware-horizon-client-2312.8.12.0.23149323-1.x86_64/usr/share/applications/vmware-view.desktop.orig vmware-horizon-client-2312.8.12.0.23149323-1.x86_64/usr/share/applications/vmware-view.desktop
--- vmware-horizon-client-2312.8.12.0.23149323-1.x86_64/usr/share/applications/vmware-view.desktop.orig	2024-01-19 09:06:57.000000000 +0100
+++ vmware-horizon-client-2312.8.12.0.23149323-1.x86_64/usr/share/applications/vmware-view.desktop	2024-02-20 11:24:55.051806303 +0100
@@ -1,10 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
 Type=Application
-Icon=/usr/share/icons/vmware-view.png
+Icon=vmware-view
 Exec=vmware-view %u
 MimeType=x-scheme-handler/vmware-view
-Categories=Application;Network;
+Categories=Network;
 Actions=about-window;
 # TRANSLATORS: Ignore this string; do not translate.
 Name=VMware Horizon Client
